docs/hardware/heating_elements.md

```diff
@@ -0,0 +1,182 @@
+
+At BrewPi we sell low watt density ($\leq$ 12 W/cm<sup>2</sup>) heating elements to prevent scorching your mash / wort. At the same time this prevents encrusting your elements and makes them really easy to clean after brewing. All elements are made from stainless steel for durability and hygenic reasons.
+
+BrewPi heating elements come in two geometries; straight and round. Straight elements are the cheaper option in general. Round elements heat your mash / wort more homogeneous, are minimally blocking your flow when whirpooling and have a even lower watt density. Technical details of the different elements can be found in the sections below.
+
+### Straight
+
+Straight BrewPi heating elements come in two types; foldback and tri-clamp. Foldback elements are the most simple elements you can find, and are the best option on a budget. Tri-clamp elements are fixed in your kettle with a tri-clamp fitting. These fittings consist of two flanges, an O-ring, and a clamp band (hence *tri*-clamp) and are the best option if you want to be able to easily remove the element from the kettle. This comes in handy when you want to change your element frequently, or want to clean it thourougly outside the kettle. 
+
+#### Foldback
+
+Details about our foldback heating elements are listed in the table below. The element length (see figure) determines the minimum kettle size (Min. ⌀) required for mounting. The element power is defined at a nominal voltage of 230V. Both foldback elements have a low watt density of 11 W/cm<sup>2</sup> and consist of 1 resistor. These elements can be connected to 1 and 2-phase outlets, and come with a 1" BSP thread. Note: do not forget to buy your element mounting kit to add grounding protection and a safety cover for the electrical wiring.
+
+![Heating elements-foldback](../images/heat-element-foldback.svg)
+
+Plaatje nog aanpassen - Is dit het origineel Elco?
+
+|  # | Length (cm) | Min. ⌀ (cm) | Power (W) | No. Resistors | Watt density (W/cm<sup>2</sup>) |
+|:--:|------------:|------------:|----------:|--------------:|--------------------------------:|
+| X* |          27 |          30 |      2800 |             1 |                            11.0 |
+| X* |          32 |          35 |      3200 |             1 |                            11.0 |
+
+X* BrewPi part. no. nog aanpassen
+
+#### Tri-clamp
+
+Details of our tri-clamp heating elements are listed below. The tri-clamp fitting is hygenic and useful when frequent removal of the element from the kettle is required. They do require a 2" tri-clamp ferrule (64mm OD) mounted on your kettle. The element length (see figure) determines the minimum kettle diameter (Min. ⌀) required for mounting. The elements are available in a big power range (2800 - 10000W), and have low watt density ($\leq$ 12 W/cm<sup>2</sup>). 2 resistor elements can be connected in a 1 and 2-phase parallel configuration, 3 resistor elements can be connected in a 1 phase parallel, and 3-phase star configuration.
```

Comment:
hygenic -> hygienic.

`The tri-clamp fitting is .... they` <- Use `it` for a back reference to a singular.